js/react/pages/admin/routes/agency/overview.jsx

```diff
@@ -40,6 +50,12 @@ export default function Overview() {
 		}
 	}, [value]);
 
+	useEffect(() => {
```

Comment:
Is this how you want to handle frontend navigation from now on? 
I previous used window for this in WishCardCreate.jsx. 
setTimeout(() => window.location.replace('/wishcards/manage'), 1000);

Should we use both or unify the redirect method for code consistency?

If we show a success toast message, we should probably let it display for a second before redirecting the page, right?

Reply:
We can use either, this is just the recommended way from the react router docs